.goreleaser.yml

```diff
@@ -0,0 +1,42 @@
+builds:
+  - <<: &build_defaults
+      id: sat
+      main: .
+      binary: sat
+      env:
+        - CGO_ENABLED=1
+      goos:
+        - linux
+        - darwin
+      goarch:
+        - amd64
+        - arm64
+      ldflags:
+        - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}
```

Comment:
Disables DWARF debugging, symbol table generation (GoReleaser defaults) and adds metadata like version, commit sha, date.